### PR TITLE
Update synthesizer test with new data

### PR DESCRIPTION
**Pull request by jepebe**
In Python 3.2 a small refactoring in Random changed the behavior of random values from seed calculation and as a result the test data is incorrect. This PR adds a new test data set for version 3.2 and up.

This PR solves #598.

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

File: python/tests/share/test_synthesizer.py [hunk 1]
```diff
@@ -11,44 +9,342 @@
     share_lib_path = os.path.join(ResTest.createSharePath("lib"))
 
     sys.path.insert(0, share_lib_path)
-    synthesizer_module =  __import__("synthesizer")
+    synthesizer_module = __import__("synthesizer")
     OilSimulator = synthesizer_module.OilSimulator
     sys.path.pop(0)
 
+if sys.version_info[0] == 3 and sys.version_info[1] >= 2:
```
Comment (pinkwah) [inline python/tests/share/test_synthesizer.py]: You can do `sys.version_info >= (3, 2)` ;)

**Comment by dotfloat** [comment]
Jenkins test this please

File: python/tests/share/test_synthesizer.py (hunk 1 above)
Comment (jepebe) [reply python/tests/share/test_synthesizer.py]: Done!

**Comment by oysteoh** [comment]
Please squash your commits , and i think we are close to approval! 👍

**Comment by jepebe** [comment]
Squished!

**Review by sondreso — APPROVED** [review]
LGTM :rocket:

**Pull request merged**